src/components/Notification/icons/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import InfoIcon from './infoIcon';
+import ErrorIcon from './errorIcon';
+import SuccessIcon from './successIcon';
+import WarningIcon from './warningIcon';
+import '../styles.css';
+
+const iconMap = {
+    info: () => <InfoIcon />,
+    error: () => <ErrorIcon />,
+    warning: () => <WarningIcon />,
+    success: () => <SuccessIcon />,
+};
+
+export default function Icon(props) {
+    const {
+        className,
+        style,
+        icon,
+    } = props;
+
+    function getClassName() {
+        return classnames('rainbow-notification_icon-container', {
+            [`rainbow-notification_icon--${icon}`]: typeof icon === 'string',
+        }, className);
+    }
+
+    if (typeof icon === 'string') {
+        return (
+            <div className={getClassName()} style={style}>
+                {iconMap[icon]()}
+            </div>
+        );
+    }
+    return (
+        <div className={getClassName()} style={style}>
```

Comment:
Since className is not needed you can use 'rainbow-notification_icon-container' className directly here. Also remove style.